legal/README.md

```diff
@@ -42,34 +43,35 @@ For personal phase, block number at which user signed agreement may be obtained
 |{repo-url}|Link to ico-contracts repo where source code of backed smart contract is stored|git@github.com:Neufund/ico-contracts.git
 |{commit-id}|Commit-id for which deployed byte code (referenced by this agreement) may be reproduced|
 |{website}|Commitment website|https://commit.neufund.org|
+|{acquisition-sc-address}|Commitment contract address|*input parameter*|
+|{signed-by-company-date}|Timestamp (UTC) at which agreement was signed by company|Commitment::currentAgreement[1]|
+|{company-address}|Ethereum address that signed agreement|Commitment::currentAgreement[0]|
+|{neumark-sc-address}|Neumark contract address|Commitment::neumark|
+|{icbm-start-date}|When public ICBM starts|Commitment::startOf(State.Public) (timestamp)|
+|{icbm-end-date}|When public ICBM ends|Commitment::startOf(State.Finished) (timestamp)|
+|{company-neumark-address}|Address where company Neumarks go|Commitment::platformWalletAddress|
+|{fork-arbiter-sc-address}|Address of fork arbiter contract|Commitment::ethereumForkArbiter|
 
 **General phase**
 
 |Tag|Description|Value|
 |---|-----------|-----|
-|{acquisition-sc-address}|Commitment contract address|*input parameter*|
 |{lockin-sc-address}|Particular LockedAccount address|*input parameter*|
-|{signed-by-company-date}|Timestamp (UTC) at which agreement was signed by company|Commitment::currentAgreement[1]|
-|{company-address}|Ethereum address that signed agreement|Commitment::currentAgreement[0]|
-|{neumark-sc-address}|Neumark contract address|Commitment::neumark|
+|{payment-token}|Token which investor uses to commit|LockedAccount::assetToken()::name|
 |{max-cap}|Maximum cap in Euro, must be converted to Ether for etherLock|Commitment:maxCapEur|
 |{min-ticket}|Minimum ticket size, must be converted to Ether for etherLock|Commitment:minTicketEur|
 |{unlock-fee-percent}|Locked account unlock penalty|LockedAccount::penaltyFraction (fraction)|
 |{fee-address}|Address where penalties go|LockedAccount::penaltyDisbursalAddress|
-|{icbm-start-date}|When public ICBM starts|Commitment::startOf(State.Public) (timestamp)|
-|{icbm-end-date}|When public ICBM ends|Commitment::startOf(State.Finished) (timestamp)|
-|{company-neumark-address}|Address where company Neumarks go|Commitment::platformWalletAddress|
-|{fork-arbiter-sc-address}|Address of fork arbiter contract|Commitment::ethereumForkArbiter|
+|{reservation-period}|Lock period in seconds|LockedAccount::lockPeriod|
 
 **Personal phase**
 
 |Tag|Description|Value|
 |---|-----------|-----|
 |{investor-address}|Address of investor for which Agreement is generated|*input parameter*|
-|{payment-token}|Token which investor uses to commit|LockedAccount::assetToken()::name|
+|{current-block-hash}|Hash of current block|*input parameter*|
```

Comment:
More precise description
Hash of the block from which all other values were taken. 
Please pay attention to take all values from that block really (not just use "latest" which will just take current block) to avoid inconsistency.